src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -92,14 +92,14 @@ int PaPulseAudio_CheckConnection( pa_context_state_t state )
             break;
 
             case PA_CONTEXT_TERMINATED:
-                PA_PULSEAUDIO_SET_LAST_HOST_ERROR( 0,
-                                                  "Connection terminated to Pulseaudio server. (PA_CONTEXT_TERMINATED)" );
+                PA_DEBUG( ("Portaudio %s: Connection terminated to Pulseaudio server. (PA_CONTEXT_TERMINATED)",
+                           __FUNCTION__) );
                 retCode = paUnanticipatedHostError;
```

Comment:
This code returns paUnanticipatedHostError.
So it should be setting the LastHostErrorInfo.

https://files.portaudio.com/docs/v19-doxydocs/portaudio_8h.html#aad573f208b60577f21d2777a7c5054e0

So it should be the way it was before.

Reply:
It's not done anymore so I assume this is fixed.